javacutil/src/main/java/org/checkerframework/javacutil/ElementUtils.java

```diff
@@ -432,8 +432,11 @@ public static boolean hasReceiver(Element element) {
             // A constructor can only have a receiver if the class it creates has an outer type.
             TypeMirror t = element.getEnclosingElement().asType();
             return TypesUtils.hasEnclosingType(t);
-        } else if (element.getKind().isField()) {
-            if (ElementUtils.isStatic(element)) {
+        } else if (element.getKind() == ElementKind.FIELD) {
+            if (ElementUtils.isStatic(element)
+                    // Artificial fields in interfaces are not marked as static, so check that
```

Comment:
I'm didn't see any easy way to mark them as static, but this could cause problems else where in the code.